lib/authorizer/digest.js

```diff
@@ -243,31 +262,54 @@ module.exports = {
             hashA1,
             hashA2,
 
+            hashAlgo,
             reqDigest,
             headerParams;
 
-        if (algorithm === MD5_SESS) {
-            A0 = crypto.MD5(username + COLON + realm + COLON + password).toString();
+        switch (algorithm) {
+            case ALGO.SHA_256:
+            case ALGO.SHA_256_SESS:
+                hashAlgo = 'sha256';
+                break;
+            case ALGO.MD5:
+            case ALGO.MD5_SESS:
+            case EMPTY:
+            case undefined:
+            case null:
+                algorithm = algorithm || ALGO.MD5;
+                hashAlgo = 'md5';
+                break;
+            case ALGO.SHA_512_256:
+            case ALGO.SHA_512_256_SESS:
+            default:
+                // Current Electron version(7.2.3) in Postman app uses OpenSSL 1.1.0
+                // which don't support `SHA-512-256`.
+                // @todo: change this when Electron is upgraded to a version which supports `SHA-512-256`.
+                throw new Error(`Unsupported digest algorithm: ${algorithm}`);
```

Comment:
Do we try the alternate algorithm / node module that you discovered?

Reply:
Will use that after doing sanity, security checks.